pkg/app/destroy.go

```diff
@@ -74,13 +75,18 @@ func destroy(clicontext *cli.Context) error {
 		}
 		ctrName = filepath.Base(buildContext)
 	}
+	c := types.Context{Runner: types.RunnerTypeDocker}
```

Comment:
Could you please get the context from the home manager? Maybe like this 

https://github.com/tensorchord/envd/blob/main/pkg/app/env_ls.go#L41:#L48

Reply:
Thank you just changed it :)